src/components/PlanRoom/RoomTables/Item.tsx

```diff
@@ -0,0 +1,184 @@
+import { useCallback, useState, SetStateAction } from 'react';
+import { Box, Textarea } from '@chakra-ui/react';
+import update from 'immutability-helper';
+import { useDrop, XYCoord, useDrag } from 'react-dnd';
+import ItemTypes from './ItemTypes';
+import { TypeItem, TableType } from './Types';
+import SaveBoard from '../SaveRoomTables/SaveBoard';
+
+let changeValueTable: Function;
+let changeValueTableBorderRadius: Function;
+let newTable: Function;
+let save: Function;
+const backgroundColor: string = '#211F1F';
+
+const Items = () => {
```

Comment:
Functions / logic contained in this file should be divided and extracted (it's to big) to proper places and couple other files.

Reply:
True, but unfortunately I was not able to export from inside the TableList functions, e.g. AddNewTable